cmd/common/st/put_object.go

```diff
@@ -16,23 +20,66 @@ const (
 	noCompressFlag     = "no-compress"
 	overwriteFlag      = "force"
 	overwriteShorthand = "f"
+	readStdinFlag 	   = "read-stdin"
+	readStdinShorthand = "s"
 )
 
+
+func openLocalFile(localPath string) (io.ReadCloser, error) {
```

Comment:
We should do only command args handling here. Let's keep the actual logic inside the handler?

Reply:
Sure, no problem, fixed